src/Query/TaskGroups.ts

```diff
@@ -1,43 +1,55 @@
 import type { Task } from '../Task';
 import type { Grouper } from './Grouper';
-import { GroupHeadings } from './GroupHeadings';
-import { IntermediateTaskGroups } from './IntermediateTaskGroups';
+import { GroupDisplayHeadingSelector } from './GroupDisplayHeadingSelector';
+import { TaskGroupingTree } from './TaskGroupingTree';
 import { TaskGroup } from './TaskGroup';
 
 /**
- * TaskGroup stores all the groups of tasks generated by any 'group by'
+ * Store all the groups of tasks generated by any 'group by'
```

Comment:
My understanding is that this is rather a storage for all the groups of tasks generated by -=EACH=- 'group by'